Common/Data/Auxiliary/TradeConditionFlags.cs

```diff
@@ -0,0 +1,140 @@
+                     "order at the price agreed upon.")]
+        StockOption = 1 << 18,
+
+        [Description("Sale of a large block of stock in such a manner that the price is not adversely affected.")]
+        Distribution = 1 << 19,
+
+        [Description("A trade where the price reported is based upon an average of the prices for transactions in a security during all or any portion of the trading day.")]
+        AveragePrice = 1 << 20,
+
+        [Description("Indicates that the trade resulted from a Market Center’s crossing session.")]
+        Cross = 1 << 21,
+
+        [Description("Indicates a regular market session trade transaction that carries a price that is significantly away from the prevailing consolidated or primary market value at the time of the transaction.")]
+        PriceVariation = 1 << 22,
+
+        [Description("To qualify as a NYSE AMEX Rule 155")]
+        Rule155 = 1 << 23,
+
+        [Description("Indicates the ‘Official’ closing value as determined by a Market Center. This transaction report will contain the market center generated closing price.")]
+        OfficialClose = 1 << 24,
+
+        [Description("A sale condition that identifies a trade based on a price at a prior point in time i.e. more than 90 seconds prior to the time of the trade report. " +
+                     "The execution time of the trade will be the time of the prior reference price.")]
+        PriorReferencePrice = 1 << 25,
+
+        [Description("Indicates the ‘Official’ open value as determined by a Market Center. This transaction report will contain the market")]
+        OfficialOpen = 1 << 26,
+
+        [Description("The CAP Election Trade highlights sales as a result of a sweep execution on the NYSE, whereby CAP orders have been elected and executed " +
+                     "outside the best price bid or offer and the orders appear as repeat trades at subsequent execution prices. " +
+                     "This indicator provides additional information to market participants that an automatic sweep transaction has occurred with repeat " +
+                     "trades as one continuous electronic transaction.")]
+        CapElection = 1 << 27,
+
+        [Description("A sale condition code that identifies a NYSE trade that has been automatically executed without the potential benefit of price improvement.")]
+        AutoExecution = 1 << 28,
+
+        [Description("Denotes whether or not a trade is exempt (Rule 611) and when used jointly with certain Sale Conditions, " +
+                     "will more fully describe the characteristics of a particular trade.")]
+        TradeThroughExempt = 1 << 29,
+
+        [Description("This flag is present in raw data, but AlgoSeek document does not describe it.")]
+        UndocumentedFlag = 1 << 30,
+
+        [Description("Denotes the trade is an odd lot less than a 100 shares.")]
+        OddLot = 1 << 31
```

Comment:
Careful that this is the last value the enum can have, else it rolls over, for example adding `Pepe = 1 << 32,`, it's okay as is today though 👍 just wondering if we will be adding new flags here

Reply:
In case we need more flags we can increase almost seamless using `int64`, e.g. 
```cs
[Flags]
public enum TrradeConditionFlags : int64
{...}
```